### ISSUE TITLE
Does not work with scss files

### ISSUE THREAD

**Issue by IcarusTalon** [poalrom/grunt-prettier#5]
grunt-prettier works great with JS files. Thanks for writing this plugin!

I really would like to use it with SCSS files as well, but it just gives an error when attempting to do that:
Running "prettier:dev" (prettier) task
Warning: Unexpected token

I have my files set up for SCSS only for testing:
prettier: {
    options: {
        singleQuote: true,
        useTabs: true
    },
    dev: {
        src: ["src/**/**.scss"]
     }

I am not completely sure, but somehow it seems that prettier.format() API call for some reason seems to expect a JS file, but I am not sure why.

Would it be possible to add SCSS(and CSS) file support in?

Thanks!

**Comment by poalrom** [comment]
Hi

Prettier doesn't support any kind of file except js. try to use other tools like [scssfmt](https://www.npmjs.com/package/scssfmt) for cli or [grunt-sass-format](https://www.npmjs.com/package/grunt-sass-format) for Grunt.

If you want to use scssfmt with grunt, I can write grunt plugin for it in a week.

**Comment by thezimmee** [comment]
@poalrom Prettier actually _does_ support SASS, as well other syntaxes (see [first paragraph of their "what is prettier?" docs](https://prettier.io/docs/en/index.html)):

```
Prettier is an opinionated code formatter with support for:

- JavaScript, including ES2017
- JSX
- Flow
- TypeScript
- CSS, Less, and SCSS
- JSON
- GraphQL
- Markdown, including GFM
```

It seems any grunt plugin should support these syntaxes as well.

**Comment by poalrom** [comment]
Yup, you are right. I should release fix at next monday

**Comment by poalrom** [comment]
Bug was fixed in version 0.3.0
